client/js/widget-manager.js

```diff
@@ -10,7 +10,7 @@
 var $ = require('jquery');
 var Widgets = require('jupyter-js-widgets');
 
-    var WidgetManager = function(kernel, msgHandler) {
+    var WidgetManager = function(kernel, kernelname, msgHandler) {
```

Comment:
I'm not sure if this is what we want to do here. The original bug was that the front end was hardcoding python3 into that field, `kernel.name`. @parente and I discussed that the back end should determine what type of kernel should be started, rather having the front end pass along the name.

However, we do need the kernel name for the widget manager. Thoughts?